AsyncSwift/AsyncSwiftApp.swift

```diff
@@ -10,9 +10,12 @@ import SwiftUI
 @main
 struct AsyncSwiftApp: App {
     @UIApplicationDelegateAdaptor(AppDelegate.self) var delegate
+	@ObservedObject var appData: AppData = AppData()
```

Comment:
```suggestion
	@ObservedObject var appData = AppData()
```

Swift는 타입추론에 능한.. ! 언어이므로 생략이 어떨까 싶어요 : )